src/terra-dev-site/example/ItemCollectionExample.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import IconAttachment from 'terra-icon/lib/icon/IconAttachment';
+import IconPerson from 'terra-icon/lib/icon/IconPerson';
+import IconAlert from 'terra-icon/lib/icon/IconAlert';
+import IconInformation from 'terra-icon/lib/icon/IconInformation';
+import ItemCollection from 'terra-clinical-item-collection';
```

Comment:
Since we want to deprecate item collection i'd rather not be pulling it into terra ui

Reply:
Fair enough, think i was using this as a base for examples or was originally including in the tutorial for a comparison. Removed now. https://github.com/cerner/terra-ui/pull/310/commits/2f0433b8a6f6fb97139694a22a60e0ad1862bf40